.github/workflows/publish_docs.yml

```diff
@@ -6,6 +6,7 @@ on:
   push:
     branches:
       - main
+      - mjj/more_docs
```

Comment:
Fixme

Reply:
Oh, I just realized you can have multiple branches listed. So main was already there. Nice one!